proto/process/agent.proto

```diff
@@ -629,6 +632,7 @@ message Connections {
 	repeated string domains = 4;
 	repeated Route routes = 5;
 	map<string, RuntimeCompilationTelemetry> compilationTelemetryByAsset = 6;
+	repeated string tags = 7;
```

Comment:
I believe that `Connections` is local to the agent, it does not get transmitted to the backend.

Reply:
You are correct. So we need something added to `CollectorConnections` as well @nplanel